### PR TITLE
Firefox 104.0.1

### PR DESCRIPTION
**Pull request by jacobgkau**
https://www.mozilla.org/en-US/firefox/104.0.1/releasenotes/

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+firefox (104.0ubuntu1) UNRELEASED; urgency=medium
```
Comment (13r0ck) [inline debian/changelog:1]:
```suggestion
firefox (104.0.1) jammy; urgency=medium
```

**Comment by n3m0-22** [comment]
All testing passed. I will wait till the conversation is resolved before approving.

[104.0.1-testing.txt](https://github.com/pop-os/packaging-firefox/files/9461807/104.0.1-testing.txt)

File: debian/changelog (hunk 1 above)
Comment (jacobgkau) [reply debian/changelog]: That's what I get for using `dch` instead of doing it manually.

**Review by n3m0-22 — APPROVED** [review]

**Review by 13r0ck — APPROVED** [review]

**Comment by herder** [comment]
Hi @n3m0-22 @13r0ck , thanks for updating this! :bow: 
I still don't see the new version in the repos, which causes some problems with my work Okta login. Do you have an ETA for when this update hits the repos?

**Comment by n3m0-22** [comment]
@herder You should be able to update Firefox now.

**Pull request merged**